stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -2336,6 +2339,19 @@ internal constructor(
             userCredentialStorage = credentialStorage
         }
 
+        public fun withApiRequestAnalyser(
```

Comment:
It seems to missed KDocs here. Also, It would be great if we could add @JvmOverloads. 😄

Reply:
I deleted this method. The `debugRequests` is enough.